lib/Doctrine/DBAL/Schema/DB2SchemaManager.php

```diff
@@ -76,10 +76,6 @@ protected function _getPortableTableColumnDefinition($tableColumn)
                 $fixed  = true;
                 break;
 
-            case 'clob':
-                $length = $tableColumn['length'];
-                break;
-
```

Comment:
This needs to be removed in order for `Db2SchemaManagerTest::testDiffListTableColumns()` to succeed.

I think this is correct because `DB2Platform` does currently not handle lengths for `blob` and `text` types, see https://github.com/doctrine/dbal/blob/b366d93c7950af1fb66ed28eea072e1400f229b5/lib/Doctrine/DBAL/Platforms/DB2Platform.php#L127-L130 and https://github.com/doctrine/dbal/blob/b366d93c7950af1fb66ed28eea072e1400f229b5/lib/Doctrine/DBAL/Platforms/DB2Platform.php#L63-L66 and this change normalizes the behavior for `blob` and `text` to work the same now.